.github/workflows/south_africa_staging.yml

```diff
@@ -0,0 +1,34 @@
+# This is a basic workflow to publish a Twilio function with Github Actions
+
+name: Serverless release to South Africa Staging.
+
+# Controls when the action will run.
+on:
+  workflow_dispatch
+  
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      # Runs a single command using the runners shell
+      - name: Install dependencies for the twilio function
+        run: npm ci
+      - name: Install twilio cli and run deploy command
+        env:
+         TWILIO_ACCOUNT_SID: ${{ secrets.SA_STG_ACCOUNT_SID }}
+         TWILIO_AUTH_TOKEN: ${{ secrets.SA_STG_AUTH_TOKEN }}
```

Comment:
Shouldn't the env variables be:
`ACCOUNT_SID` instead of `TWILIO_ACCOUNT_SID`
`AUTH_TOKEN` instead of `TWILIO_AUTH_TOKEN`

That's how it is in the docs, but it may be outdated.

Reply:
Just checked and I confirm, we are using `ACCOUNT_SID` and `AUTH_TOKEN` like stated in the docs.

Reply:
Thanks for checking this. I will correct it to use the right format.